Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/IObjectSerializer.cs

```diff
@@ -10,19 +10,19 @@ namespace Microsoft.Toolkit.Uwp.Helpers
     public interface IObjectSerializer
     {
         /// <summary>
-        /// Serialize an object into a string.
+        /// Serialize an object into a string. It is recommended to use strings as the final format for objects if you plan to use the <see cref="BaseObjectStorageHelper.SaveFileAsync{T}(string, T)"/> method.
```

Comment:
@vgromfeld which part in particular? It only highlights the line, so I'm not sure what 'this part' refers to?

Reply:
Indeed 😅. It was about the "Serialize an object into a string". 
With the last change it should becomes something like "Serialize an object into an object (which should be a string)" 😀

Reply:
I'll munch on that a bit, we'll have a lot of docs/examples of implementations for this class to support any common serializer, so don't think this will get looked at too much beyond that.